setup.cfg

```diff
@@ -47,7 +47,13 @@ install_requires =
     napari-tools-menu
     napari-skimage-regionprops>=0.3.1
     hdbscan
-    joblib
+    qtpy
```

Comment:
```suggestion
```
I strongly discourage adding this dependency because of licensing reasons. Also we do not use any functions from this package.

Reply:
No, wait I take this back. It's the right dependency  :-)